packages/api/tsconfig.dist.json

```diff
@@ -10,6 +10,7 @@
     "outDir": "dist",
     "declarationDir": "@types",
     "paths": {
+      "loot-core/src/*": ["./loot-core/*"],
```

Comment:
After running a `yarn build` within `package/api` we had some imports in the .d.ts files that were still referencing `src`.